example/vaultHelpers/init.sh

```diff
@@ -0,0 +1,29 @@
+apk add jq curl
```

Comment:
Can you please add comments before lines here so devs who need help can understand what they copy-paste? :)